doc/releases/changelog-0.27.0.md

```diff
@@ -4,57 +4,226 @@
 
 <h3>New features since last release</h3>
 
-<h4>(TODO: title) JAX JIT</h4>
+Missing entries:
+
+* Functionality is added to estimate the number of measurements required to compute an expectation
+  value with a target error and estimate the error in computing an expectation value with a given 
+  number of measurements.
+  [(#3000)](https://github.com/PennyLaneAI/pennylane/pull/3000)
+
+* [(#2886)](https://github.com/PennyLaneAI/pennylane/pull/2886)
+* [(#3052)](https://github.com/PennyLaneAI/pennylane/pull/3052)
+* [(#3041)](https://github.com/PennyLaneAI/pennylane/pull/3041)
+* [(#3090)](https://github.com/PennyLaneAI/pennylane/pull/3090)
+* [(#3069)](https://github.com/PennyLaneAI/pennylane/pull/3069)
+* [(#3137)](https://github.com/PennyLaneAI/pennylane/pull/3137)
+* [(#3127)](https://github.com/PennyLaneAI/pennylane/pull/3127)
+* [(#3121)](https://github.com/PennyLaneAI/pennylane/pull/3121)
+* [(#3099)](https://github.com/PennyLaneAI/pennylane/pull/3099)
+* [(#3098)](https://github.com/PennyLaneAI/pennylane/pull/3098)
+* [(#3095)](https://github.com/PennyLaneAI/pennylane/pull/3095)
+* [(#3091)](https://github.com/PennyLaneAI/pennylane/pull/3091)
+* [(#3176)](https://github.com/PennyLaneAI/pennylane/pull/3176)
+* [(#3170)](https://github.com/PennyLaneAI/pennylane/pull/3170)
+* [(#3157)](https://github.com/PennyLaneAI/pennylane/pull/3157)
+* [(#3195)](https://github.com/PennyLaneAI/pennylane/pull/3195)
+* [(#3194)](https://github.com/PennyLaneAI/pennylane/pull/3194)
+* [(#3184)](https://github.com/PennyLaneAI/pennylane/pull/3184)
+* [(#3249)](https://github.com/PennyLaneAI/pennylane/pull/3249)
+* [(#3267)](https://github.com/PennyLaneAI/pennylane/pull/3267)
+* [(#3234)](https://github.com/PennyLaneAI/pennylane/pull/3234)
+* [(#3233)](https://github.com/PennyLaneAI/pennylane/pull/3233)
+* [(#3232)](https://github.com/PennyLaneAI/pennylane/pull/3232)
+* [(#3223)](https://github.com/PennyLaneAI/pennylane/pull/3223)
+* [(#3222)](https://github.com/PennyLaneAI/pennylane/pull/3222)
```

Comment:
These were marked with a *** in [this spreadsheet](https://docs.google.com/spreadsheets/d/1tYaVRHDqsrzSFZviJ0VRPDtR4BiOfrYhC8_LjJYK2TU/edit#gid=1295754102)